### DISCUSSION TITLE
How can we get the script without using NPM?

### DISCUSSION THREAD

**Discussion by AckItsMe** [chartjs/chartjs-plugin-zoom#912] [General]
I can pull the code from here but there's no build .js file.

I don't use node.js so I don't have npm.

How else can I get the script?

Thanks!

**Comment by AckItsMe** [comment]
Never mind... I was able to locate it on the https://github.com/chartjs/chartjs-plugin-zoom/releases page.

**Reply by kurkle** [reply]
NPM packages are generally available through various content delivery networks, for example:

https://www.jsdelivr.com/package/npm/chartjs-plugin-zoom
